ilastik/shell/gui/ilastikShell.py

```diff
@@ -1832,6 +1834,8 @@ def saveAs():
 
                 try:
                     self.projectManager.saveProjectAs(newPath)
+                    workflowDisplayName = self.projectManager.workflow.workflowDisplayName
+                    self._setRecentlyOpenedList(newPath, workflowDisplayName)
```

Comment:
a bit of a nitpick, but technically the try-except block only makes sense for `self.projectManager.saveProjectAs(newPath)` (as only `ProjectManager.SaveError` is caught here). Anything else will raise. So I'd put the code after the except block.

Reply:
Thank you very much for the review! My pleasure contributing :)

Initially I also put it after the except block, and then I changed it, because I thought that if there is a save error, the list shouldn't be updated. Please correct me if I am wrong.

Reply:
yes, you're totally right :)